Cargo.lock

```diff
@@ -585,6 +688,36 @@ dependencies = [
  "typenum",
 ]
 
+[[package]]
+name = "curl"
+version = "0.4.44"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "509bd11746c7ac09ebd19f0b17782eae80aadee26237658a6b4808afb5c11a22"
+dependencies = [
+ "curl-sys",
+ "libc",
+ "openssl-probe",
+ "openssl-sys",
+ "schannel",
+ "socket2",
+ "winapi",
+]
+
+[[package]]
+name = "curl-sys"
+version = "0.4.65+curl-8.2.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "961ba061c9ef2fe34bbd12b807152d96f0badd2bebe7b90ce6c8c8b7572a0986"
+dependencies = [
+ "cc",
+ "libc",
+ "libz-sys",
+ "openssl-sys",
+ "pkg-config",
+ "vcpkg",
+ "winapi",
+]
```

Comment:
Are there any options for Rust HTTP clients?

Reply:
I have added a commit that removes `curl` in favor of the only other available HTTPS backend: `reqwest`.

Reply:
We should switch from `crates-index` to `tame-index` as covered [on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/146229-wg-secure-code/topic/removing.20OpenSSL.20from.20.60cargo.20audit.60), following the same switch in `cargo-deny`.

Reply:
Ah, great! Just today I learned about `tame-index` independently of that and it looks like it's the way to go!

I am happy to help with/implement the switch from `git2` to `gix` if you want me to, please feel free to ping me in GitHub or on zulip.